netrd/dynamics/sherrington_kirkpatrick.py

```diff
@@ -0,0 +1,89 @@
+import networkx as nx
+import numpy as np
+
+class SherringtonKirkpatrickIsing:
+    """
+    Kinetic Ising model from 
```

Comment:
Can you move this docstring into the `simulate` docstring? Check some of the distance files for a comparison.